src/WebJobs.Script/Rpc/Configuration/WorkerConfigFactory.cs

```diff
@@ -220,5 +223,37 @@ internal string GetExecutablePathForJava(string defaultExecutablePath)
                 return Path.GetFullPath(Path.Combine(javaHome, "bin", defaultExecutablePath));
             }
         }
+
+        internal string HydrateWorkerPath(string workerPath, string language)
+        {
+            if (string.IsNullOrEmpty(workerPath))
+            {
+                return null;
+            }
+
+            string os = _systemRuntimeInformation.GetOSPlatform().ToString();
+
+            string architecture = _systemRuntimeInformation.GetOSArchitecture().ToString();
```

Comment:
values of Architechture as documented here: https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.architecture?view=netframework-4.8 are 0 through 3. Do you plan to include folders matching 0, 1,2 and 3?

Reply:
I wasn't planning on adding support for Arm and Arm64 at least in the first pass.